ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestSteps.js

```diff
@@ -0,0 +1,106 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Icon, Segment, Step } from 'semantic-ui-react';
+import { getDisplayVal, invenioConfig } from '@config/invenioConfig';
+import { LoanIcon } from '@pages/backoffice/components';
+
+const toLabel = status => {
+  return getDisplayVal('illBorrowingRequests.statuses', status);
+};
+
+const isBefore = (currentStatus, status) => {
+  const orderedStatuses =
+    invenioConfig.illBorrowingRequests.orderedValidStatuses;
+  return (
+    orderedStatuses.indexOf(currentStatus) < orderedStatuses.indexOf(status)
+  );
+};
+
+const Pending = ({ currentStatus, disabled }) => {
```

Comment:
potentially they could be a component if you think it makes sense (also the ones in acquisition)

Reply:
done, I still prefer to keep Order/BorrowingRequest separated even if they are similar, if ok with you.